apps/rush-lib/src/start.ts

```diff
@@ -1,6 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
+console.time('total time');
 import { Rush } from './api/Rush';
 
-Rush.launch(Rush.version, { isManaged: false });
+Rush.launch(Rush.version, { isManaged: false }, () => {
+  console.timeEnd('total time');
```

Comment:
Were you intending to actually print this every time the Rush command is invoked?  Seems a bit distracting. If you want to keep this feature in the code, maybe it should be opt-in via a special CLI switch like `--measure-execution-time`.

For my own tests, I simply used PowerShell's `Measure-Command`.

Reply:
I will remove this.